src/app/App.js

```diff
@@ -682,6 +682,33 @@ class App extends EventEmitter {
 	}
 
 	/**
+	 * Maybe restore redirected path hash in case both the current path and
+	 * the given path are the same.
+	 * @param {!string} path Path before navigation.
+	 * @param {!string} redirectPath Path after navigation.
+	 * @param {!string} hash Hash to be added to the path.
+	 * @return {!string} Returns the path with the hash restored.
+	 */
+	maybeRestoreRedirectPathHash_(path, redirectPath, hash) {
+		if (redirectPath === utils.getUrlPathWithoutHash(path)) {
+			return redirectPath + hash;
+		}
+		return redirectPath;
+	}
+
+	/**
+	 * Maybe update scroll position in history state to anchor on path.
+	 * @param {!string} path Path containing anchor
+	 */
+	maybeUpdateScrollPositionState_() {
+		var hash = globals.window.location.hash
```

Comment:
Fixed. I was running gulp format. For some reason I though it would call the linter and warn me about that. Thx!